src/components/AdvertiseBar.tsx

```diff
@@ -0,0 +1,8 @@
+const AdvertiseBar = () => {
```

Comment:
I don't think this should be a component

Reply:
![image](https://github.com/PiedTeam/NikeCloneTraining-FE-Project/assets/137786692/7eca7c47-5520-477c-b902-5052202d2f98)
I create this component for later, as it is independent from navbar and just appear on several pages